packages/plugin-apex/src/commands/force/apex/test/run.ts

```diff
@@ -374,4 +357,25 @@ export default class Run extends SfdxCommand {
     const hint = messages.getMessage('apexTestReportFormatHint', [reportArgs]);
     return hint;
   }
+
+  /**
+   * Handles special exceptions where we don't want to return early
+   * with the testRunId.
+   * */
+  private shouldImmediatelyReturn(): boolean {
+    if (this.flags.resultformat !== undefined) {
+      return false;
+    }
+
+    //when the user has explictly asked to wait for results, but didn't give a format
```

Comment:
nit: do we not have prettier on this repo?  I'd expect a space between the // when. Also no space on line 364 
`* */` -> `**/`

Reply:
Hmm...  I'm pretty sure we do have a prettier rule for this, as I remember getting a lint error about this once.  Perhaps the rule is in salesforcedx-vscode and not salesforcedx-apex.